boards/arm/efm32wg_stk3800/board.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2017 Christian Taedcke
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <board.h>
+#include <gpio.h>
+
+static int efm32wg_stk3800_init(struct device *dev)
+{
+	struct device *bce_dev; /* Board Controller Enable Gpio Device */
+
+	ARG_UNUSED(dev);
+
+	/* Enable the board controller to be able to use the serial port */
+	bce_dev = device_get_binding(BC_ENABLE_GPIO_NAME);
```

Comment:
Could this block be done in the device tree somehow -> set a gpio pin to 1 constantly on bootup?

Reply:
if not it dev tree, you would need to check the return of device_get_binding here

Reply:
done